src/Microsoft.AspNet.Mvc.Core/ApplicationModels/DefaultControllerModelBuilder.cs

```diff
@@ -31,11 +34,13 @@ public class DefaultControllerModelBuilder : IControllerModelBuilder
         public DefaultControllerModelBuilder(
             IActionModelBuilder actionModelBuilder, 
             ILoggerFactory loggerFactory, 
```

Comment:
I think you can ask for `ILogger<DefaultControllerModelBuilder>` as a service in the constructor now, if you don't want to get the `ILoggerFactory` and call Create